### PR TITLE
End session endpoint

### PR DESCRIPTION
**Pull request by gfioretti**
- Implements End Session endpoint according to https://appium.io/docs/en/commands/session/delete/

### CODE REVIEW

**Comment by KazuCocoa** [comment]
The CI fail looks the new file hasn't added in compile sources in AppiumSwiftClient target

**Comment by gfioretti** [comment]
@KazuCocoa I noticed this after trying a couple of times, but couldn't figure out why. Would you have any idea why is this happening?

File: AppiumSwiftClient/command/W3C/EndSession.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+        if statusCode == 200 {
+            return ""
+        } else if statusCode == 404 {
+            print("Status Code \(statusCode): No Session with id \(sessionId) found?")
+            print(returnValue)
+            return ""
+        } else {
+            return ""
+        }
```
Comment (KazuCocoa) [inline AppiumSwiftClient/command/W3C/EndSession.swift:27]: Here could simplify as below
```swift
if statusCode == 404 {
   print(".....")
}
return ""
```

File: AppiumSwiftClient/command/W3C/EndSession.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    func commandUrl(with sessionId: Session.Id, and _: Element.Id = "") -> W3CCommands.CommandPath {
+        return W3CCommands().url(for: W3CCommands.deleteSession, with: sessionId)
+    }
+
+    func generateBodyData() -> Data {
+        let invalidJson = "invalid JSON"
```
Comment (KazuCocoa) [inline AppiumSwiftClient/command/W3C/EndSession.swift:35]: Perhaps here can be `return "{}".data(using: .utf8)!`

File: AppiumSwiftClient/command/W3C/EndSession.swift (hunk 1 above)
Comment (gfioretti) [reply AppiumSwiftClient/command/W3C/EndSession.swift:27]: Makes total sense

File: AppiumSwiftClient.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -914,7 +910,7 @@
 				OTHER_LDFLAGS = "-ObjC";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```
Comment (KazuCocoa) [inline AppiumSwiftClient.xcodeproj/project.pbxproj:913]: 👍 
I'm fine to bump swift version up.
Appium server supports Xcode 10 (I mean Swift 4.2), but no reason to support the same version

**Pull request merged**